content/en/docs/contribute/analytics.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Viewing site analytics
+content_type: concept
+weight: 100
+card:
+  name: contribute
+  weight: 100
+---
+
+<!-- overview -->
+
+This page contains information about the kubernetes.io analytics dashboard.
+
+
+<!-- body -->
+
+[View the dashboard.](https://datastudio.google.com/u/0/reporting/fede2672-b2fd-402a-91d2-7473bdb10f04/page/567IC/edit)
+
+This dashboard is built using Google Data Studio and shows information collected on kubernetes.io using Google Analytics.
+
+### Using the dashboard
+
+By default, the dashboard will show all collected analytics for the past 30 days. Use the date selector to see data from a different date range. Other filtering options allow you to view data based on user location, the device used to access the site, the translation of the docs used, and more.
+
+ If you notice an issue with this dashboard, or would like to request any improvements, please open an issue.
```

Comment:
Optionally, make “open an issue” a hyperlink to https://github.com/kubernetes/website/issues/new/choose

Reply:
added!